pinterest/teletraan
# Adding Conditional for sidecars to remove stage type banner

The purpose of this PR is to remove the stage type banner from sidecar stages (stages where system priority is not null)

Test Plan:
1. update a stage system priority of 1
2. Navigate to the env and confirm that the banner does no show on the landing page, the config page or the deploy modal
